doc/man3/OSSL_sk_X509_add_cert.pod

```diff
@@ -0,0 +1,50 @@
+=pod
+
+=head1 NAME
+
+OSSL_sk_X509_add_cert,
+OSSL_sk_X509_add_certs -
+X509 certificate list handling functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/x509.h>
+
+ int OSSL_sk_X509_add_cert(STACK_OF(X509) *sk, X509 *cert,
+                           int up, int no_dup, int prepend);
+ int OSSL_sk_X509_add_certs(STACK_OF(X509) *sk, STACK_OF(X509) *certs, int up,
+                            int no_self_issued, int no_dups, int prepend);
```

Comment:
I wonder whether the list of flags at the end of these functions might be better replaced by a single "flags" parameter? This gives us the flexibility to add new ones if we think of them. I also think it will be more readable.

Reply:
Good idea! I agree that this both adds flexibility and readability. Done.